extensions/amp-story/1.0/amp-story.css

```diff
@@ -176,6 +176,22 @@ amp-story-page[active] {
   z-index: 1 !important;
 }
 
+amp-sidebar {
+  background-color: #ffffff !important;
+  max-width: 280px !important;
+  transition: opacity 0.3s cubic-bezier(0.0, 0.0, 0.2, 1) !important;
```

Comment:
Can we do a slide in/out transition instead, similar to what we have for the share menu?